--- phing/phing/etc/phpunit-frames.xsl~	2026-01-10 16:20:23.930978915 +0100
+++ phing/phing/etc/phpunit-frames.xsl	2026-01-10 17:04:15.899052799 +0100
@@ -79,7 +79,7 @@
 <xsl:template name="package">
     <xsl:param name="name"/>
     <xsl:variable name="package.dir">
-        <xsl:if test="not($name = '')"><xsl:value-of select="translate($name,'.','/')"/></xsl:if>
+        <xsl:if test="not($name = '')"><xsl:value-of select="translate(translate($name,'.','/'),'\','/')"/></xsl:if>
         <xsl:if test="$name = ''">.</xsl:if>
     </xsl:variable>
     <!--Processing package <xsl:value-of select="@name"/> in <xsl:value-of select="$output.dir"/> -->
@@ -537,7 +537,13 @@
             <xsl:with-param name="path"><xsl:value-of select="substring-after($path,'/')"/></xsl:with-param>
         </xsl:call-template>
     </xsl:if>
-    <xsl:if test="not(contains($path,'.')) and not(contains($path,'/')) and not($path = '')">
+    <xsl:if test="contains($path,'\')">
+        <xsl:text>../</xsl:text>
+        <xsl:call-template name="path">
+            <xsl:with-param name="path"><xsl:value-of select="substring-after($path,'\')"/></xsl:with-param>
+        </xsl:call-template>
+    </xsl:if>
+    <xsl:if test="not(contains($path,'.')) and not(contains($path,'/')) and not(contains($path,'\')) and not($path = '')">
         <xsl:text>../</xsl:text>
     </xsl:if>
 </xsl:template>
