diff --git a/node_modules/@http-util/status-i18n/index.d.ts b/node_modules/@http-util/status-i18n/index.d.ts
index b4bdcc3..50d5ab6 100644
--- a/node_modules/@http-util/status-i18n/index.d.ts
+++ b/node_modules/@http-util/status-i18n/index.d.ts
@@ -3,4 +3,5 @@ declare module '@http-util/status-i18n' {
     statusCode: number | undefined,
     locale: string
   ): string | undefined
+  export default status
 }
