diff --git a/node_modules/@nextcloud/l10n/dist/chunks/translation-DoG5ZELJ.mjs b/node_modules/@nextcloud/l10n/dist/chunks/translation-DoG5ZELJ.mjs
index aa0aa71..ed84d97 100644
--- a/node_modules/@nextcloud/l10n/dist/chunks/translation-DoG5ZELJ.mjs
+++ b/node_modules/@nextcloud/l10n/dist/chunks/translation-DoG5ZELJ.mjs
@@ -147,7 +147,7 @@ function translate(app, text, placeholdersOrNumber, optionsOrNumber, options) {
       }
     });
   };
-  const bundle = options?.bundle ?? getAppTranslations(app);
+  const bundle = allOptions?.bundle ?? getAppTranslations(app);
   let translation = bundle.translations[text] || text;
   translation = Array.isArray(translation) ? translation[0] : translation;
   if (typeof vars === "object" || number !== void 0) {
@@ -185,7 +185,8 @@ async function loadTranslations(appName, callback) {
   }
   let response;
   try {
-    const url = generateFilePath(appName, "l10n", getLanguage() + ".json");
+    const language = getLanguage().replace('-', '_');
+    const url = generateFilePath(appName, "l10n", language + ".json");
     response = await fetch(url);
   } catch (error) {
     throw new Error("Network error", { cause: error });
diff --git a/node_modules/@nextcloud/l10n/dist/chunks/translation-oV9XBOJH.cjs b/node_modules/@nextcloud/l10n/dist/chunks/translation-oV9XBOJH.cjs
index 85e38e0..02c5d37 100644
--- a/node_modules/@nextcloud/l10n/dist/chunks/translation-oV9XBOJH.cjs
+++ b/node_modules/@nextcloud/l10n/dist/chunks/translation-oV9XBOJH.cjs
@@ -151,7 +151,7 @@ function translate(app, text, placeholdersOrNumber, optionsOrNumber, options) {
       }
     });
   };
-  const bundle = options?.bundle ?? getAppTranslations(app);
+  const bundle = allOptions?.bundle ?? getAppTranslations(app);
   let translation = bundle.translations[text] || text;
   translation = Array.isArray(translation) ? translation[0] : translation;
   if (typeof vars === "object" || number !== void 0) {
