diff --git a/node_modules/@nextcloud/vue/dist/Components/NcActionLink.cjs b/node_modules/@nextcloud/vue/dist/Components/NcActionLink.cjs
index f612f4b..384ce5d 100644
--- a/node_modules/@nextcloud/vue/dist/Components/NcActionLink.cjs
+++ b/node_modules/@nextcloud/vue/dist/Components/NcActionLink.cjs
@@ -64,7 +64,7 @@ const _sfc_main = {
 };
 var _sfc_render = function render() {
   var _vm = this, _c = _vm._self._c;
-  return _c("li", { staticClass: "action", attrs: { "role": _vm.isInSemanticMenu && "presentation" } }, [_c("a", { staticClass: "action-link focusable", attrs: { "download": _vm.download, "href": _vm.href, "aria-label": _vm.ariaLabel, "target": _vm.target, "title": _vm.title, "rel": "nofollow noreferrer noopener", "role": _vm.isInSemanticMenu && "menuitem" }, on: { "click": _vm.onClick } }, [_vm._t("icon", function() {
+  return _c("li", { staticClass: "action", attrs: { "role": _vm.isInSemanticMenu && "presentation" } }, [_c("a", { staticClass: "action-link focusable", attrs: { "download": _vm.download, "href": _vm.href, "aria-label": _vm.ariaLabel, "target": _vm.target, "title": _vm.title, "rel": _vm.target.startsWith('_') ? "nofollow noreferrer noopener" : "nofollow", "role": _vm.isInSemanticMenu && "menuitem" }, on: { "click": _vm.onClick } }, [_vm._t("icon", function() {
     return [_c("span", { staticClass: "action-link__icon", class: [_vm.isIconUrl ? "action-link__icon--url" : _vm.icon], style: { backgroundImage: _vm.isIconUrl ? `url(${_vm.icon})` : null }, attrs: { "aria-hidden": "true" } })];
   }), _vm.name ? _c("span", { staticClass: "action-link__longtext-wrapper" }, [_c("strong", { staticClass: "action-link__name" }, [_vm._v(" " + _vm._s(_vm.name) + " ")]), _c("br"), _c("span", { staticClass: "action-link__longtext", domProps: { "textContent": _vm._s(_vm.text) } })]) : _vm.isLongText ? _c("span", { staticClass: "action-link__longtext", domProps: { "textContent": _vm._s(_vm.text) } }) : _c("span", { staticClass: "action-link__text" }, [_vm._v(_vm._s(_vm.text))]), _vm._e()], 2)]);
 };
diff --git a/node_modules/@nextcloud/vue/dist/Components/NcActionLink.mjs b/node_modules/@nextcloud/vue/dist/Components/NcActionLink.mjs
index 4427f1d..0cadf27 100644
--- a/node_modules/@nextcloud/vue/dist/Components/NcActionLink.mjs
+++ b/node_modules/@nextcloud/vue/dist/Components/NcActionLink.mjs
@@ -63,7 +63,7 @@ const _sfc_main = {
 };
 var _sfc_render = function render() {
   var _vm = this, _c = _vm._self._c;
-  return _c("li", { staticClass: "action", attrs: { "role": _vm.isInSemanticMenu && "presentation" } }, [_c("a", { staticClass: "action-link focusable", attrs: { "download": _vm.download, "href": _vm.href, "aria-label": _vm.ariaLabel, "target": _vm.target, "title": _vm.title, "rel": "nofollow noreferrer noopener", "role": _vm.isInSemanticMenu && "menuitem" }, on: { "click": _vm.onClick } }, [_vm._t("icon", function() {
+  return _c("li", { staticClass: "action", attrs: { "role": _vm.isInSemanticMenu && "presentation" } }, [_c("a", { staticClass: "action-link focusable", attrs: { "download": _vm.download, "href": _vm.href, "aria-label": _vm.ariaLabel, "target": _vm.target, "title": _vm.title, "rel": _vm.target.startsWith('_') ? "nofollow noreferrer noopener" : "nofollow", "role": _vm.isInSemanticMenu && "menuitem" }, on: { "click": _vm.onClick } }, [_vm._t("icon", function() {
     return [_c("span", { staticClass: "action-link__icon", class: [_vm.isIconUrl ? "action-link__icon--url" : _vm.icon], style: { backgroundImage: _vm.isIconUrl ? `url(${_vm.icon})` : null }, attrs: { "aria-hidden": "true" } })];
   }), _vm.name ? _c("span", { staticClass: "action-link__longtext-wrapper" }, [_c("strong", { staticClass: "action-link__name" }, [_vm._v(" " + _vm._s(_vm.name) + " ")]), _c("br"), _c("span", { staticClass: "action-link__longtext", domProps: { "textContent": _vm._s(_vm.text) } })]) : _vm.isLongText ? _c("span", { staticClass: "action-link__longtext", domProps: { "textContent": _vm._s(_vm.text) } }) : _c("span", { staticClass: "action-link__text" }, [_vm._v(_vm._s(_vm.text))]), _vm._e()], 2)]);
 };
diff --git a/node_modules/@nextcloud/vue/dist/Components/NcModal.cjs b/node_modules/@nextcloud/vue/dist/Components/NcModal.cjs
index 843ddc4..0799e6e 100644
--- a/node_modules/@nextcloud/vue/dist/Components/NcModal.cjs
+++ b/node_modules/@nextcloud/vue/dist/Components/NcModal.cjs
@@ -559,7 +559,11 @@ const __default__ = {
         trap.deactivate();
       }
       this.focusTrap = focusTrap$1.createFocusTrap([contentContainer, ...this.additionalTrapElements], options);
-      this.focusTrap.activate();
+      try {
+        this.focusTrap.activate();
+      } catch(error) {
+        console.error('Unable to activate focus trap', { error });
+      }
     },
     clearFocusTrap() {
       if (!this.focusTrap) {
diff --git a/node_modules/@nextcloud/vue/dist/Components/NcModal.mjs b/node_modules/@nextcloud/vue/dist/Components/NcModal.mjs
index 1dff045..8187140 100644
--- a/node_modules/@nextcloud/vue/dist/Components/NcModal.mjs
+++ b/node_modules/@nextcloud/vue/dist/Components/NcModal.mjs
@@ -556,7 +556,11 @@ const __default__ = {
         trap.deactivate();
       }
       this.focusTrap = createFocusTrap([contentContainer, ...this.additionalTrapElements], options);
-      this.focusTrap.activate();
+      try {
+        this.focusTrap.activate();
+      } catch(error) {
+        console.error('Unable to activate focus trap', { error });
+      }
     },
     clearFocusTrap() {
       if (!this.focusTrap) {
diff --git a/node_modules/@nextcloud/vue/dist/assets/NcAppSidebar-Dun97zcC.css b/node_modules/@nextcloud/vue/dist/assets/NcAppSidebar-Dun97zcC.css
index ffea28e..5174238 100644
--- a/node_modules/@nextcloud/vue/dist/assets/NcAppSidebar-Dun97zcC.css
+++ b/node_modules/@nextcloud/vue/dist/assets/NcAppSidebar-Dun97zcC.css
@@ -173,7 +173,7 @@
   flex-direction: row;
 }
 .app-sidebar .app-sidebar-header--compact.app-sidebar-header--with-figure .app-sidebar-header__info .app-sidebar-header__figure[data-v-478c9d0b] {
-  --figure-size: calc($desc-height + var(--app-sidebar-padding));
+  --figure-size: calc(8px + var(--app-sidebar-padding));
   z-index: 2;
   width: var(--figure-size);
   height: var(--figure-size);
