diff --git a/node_modules/@types/chosen-js/index.d.ts b/node_modules/@types/chosen-js/index.d.ts
index 877602f..bf0a28a 100644
--- a/node_modules/@types/chosen-js/index.d.ts
+++ b/node_modules/@types/chosen-js/index.d.ts
@@ -130,13 +130,13 @@ declare namespace Chosen {
     };
 }
 
-interface JQuery {
-    chosen(options?: Chosen.Options | "destroy"): JQuery;
+interface JQuery<TElement = HTMLElement> {
+    chosen(options?: Chosen.Options | "destroy"): JQuery<TElement>;
 
     /**
      * Chosen triggers the standard DOM event whenever a selection is made (it also sends a selected or deselected parameter that tells you which option was changed).
      */
-    on(events: "change", handler: (eventObject: JQuery.TriggeredEvent, args: Chosen.SelectedData) => any): JQuery;
+    on(events: "change", handler: (eventObject: JQuery.TriggeredEvent, args: Chosen.SelectedData) => any): JQuery<TElement>;
 
     /**
      * * `chosen:ready` Triggered after Chosen has been fully instantiated.
@@ -145,7 +145,7 @@ interface JQuery {
      * * `chosen:hiding_dropdown` Triggered when Chosen’s dropdown is closed.
      * * `chosen:no_results` Triggered when a search returns no matching results.
      */
-    on(events: Chosen.OnEvent, handler: (eventObject: JQuery.TriggeredEvent) => any): JQuery;
+    on(events: Chosen.OnEvent, handler: (eventObject: JQuery.TriggeredEvent) => any): JQuery<TElement>;
 
     /**
      * * `chosen:updated` This event should be triggered whenever Chosen’s underlying select element changes (such as a change in selected options).
@@ -153,5 +153,5 @@ interface JQuery {
      * * `chosen:open` This event activates Chosen and also displays the search results.
      * * `chosen:close` This event deactivates Chosen and hides the search results.
      */
-    trigger(eventType: Chosen.TriggerEvent): JQuery;
+    trigger(eventType: Chosen.TriggerEvent): JQuery<TElement>;
 }
