diff --git a/node_modules/@types/selectize/index.d.ts b/node_modules/@types/selectize/index.d.ts
index e9cd923..99c89f5 100644
--- a/node_modules/@types/selectize/index.d.ts
+++ b/node_modules/@types/selectize/index.d.ts
@@ -504,7 +504,7 @@ declare namespace Selectize {
         /**
          * Re-renders the selected item lists.
          */
-        refreshItems(): void;
+        refreshItems(silent?: boolean): void;
 
         // Optgroups
         // ------------------------------------------------------------------------------------------------------------
@@ -683,9 +683,5 @@ declare namespace Selectize {
 }
 
 interface JQuery {
-    selectize(options?: Selectize.IOptions<any, any>): JQuery;
-}
-
-interface HTMLElement {
-    selectize: Selectize.IApi<any, any>;
+    selectize<T extends string, U extends string>(options?: Selectize.IOptions<T, U>): JQuery;
 }
