diff --git a/node_modules/bootstrap/js/dist/tooltip.js b/node_modules/bootstrap/js/dist/tooltip.js
index 033dd7b..1270cc7 100644
--- a/node_modules/bootstrap/js/dist/tooltip.js
+++ b/node_modules/bootstrap/js/dist/tooltip.js
@@ -252,8 +252,10 @@
         if (!this._isHovered) {
           this._disposePopper();
         }
-        this._element.removeAttribute('aria-describedby');
-        EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN));
+        if (this._element) {
+          this._element.removeAttribute('aria-describedby');
+          EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN));
+        }
       };
       this._queueCallback(complete, this.tip, this._isAnimated());
     }
@@ -459,7 +461,7 @@
       this._timeout = setTimeout(handler, timeout);
     }
     _isWithActiveTrigger() {
-      return Object.values(this._activeTrigger).includes(true);
+      return Object.values(this._activeTrigger || {}).includes(true);
     }
     _getConfig(config) {
       const dataAttributes = Manipulator.getDataAttributes(this._element);
diff --git a/node_modules/bootstrap/scss/_tooltip-with-deps.scss b/node_modules/bootstrap/scss/_tooltip-with-deps.scss
new file mode 100644
index 0000000..351f8cf
--- /dev/null
+++ b/node_modules/bootstrap/scss/_tooltip-with-deps.scss
@@ -0,0 +1,7 @@
+@import 'bootstrap/scss/functions';
+@import 'bootstrap/scss/variables';
+@import 'bootstrap/scss/mixins/border-radius';
+@import 'bootstrap/scss/mixins/deprecate';
+@import 'bootstrap/scss/mixins/reset-text';
+@import 'bootstrap/scss/vendor/rfs';
+@import 'bootstrap/scss/tooltip';
