diff --git a/node_modules/eslint-import-resolver-jest/lib/index.js b/node_modules/eslint-import-resolver-jest/lib/index.js
index 6a46e2f..0eb41d5 100644
--- a/node_modules/eslint-import-resolver-jest/lib/index.js
+++ b/node_modules/eslint-import-resolver-jest/lib/index.js
@@ -77,6 +77,9 @@ function getJestConfig(config                  = {}, file      )             {
     } catch (e) {
       throw new Error(`jestConfigFile not found in ${configFilePath}`);
     }
+    if (jestConfig.__esModule && jestConfig.default) {
+      jestConfig = jestConfig.default;
+    }
   } else {
     const packageJson = require(path.resolve(root, 'package.json'));
     jestConfig = packageJson.jest;
@@ -86,7 +89,6 @@ function getJestConfig(config                  = {}, file      )             {
   }
 
   jestConfig.importResolverProjectRoot = root;
-
   return { ...JEST_DEFAULT_CONFIG, ...jestConfig };
 }
 
