diff --git a/node_modules/jquery-datetimepicker/build/jquery.datetimepicker.full.js b/node_modules/jquery-datetimepicker/build/jquery.datetimepicker.full.js
index 7245608..451fabb 100644
--- a/node_modules/jquery-datetimepicker/build/jquery.datetimepicker.full.js
+++ b/node_modules/jquery-datetimepicker/build/jquery.datetimepicker.full.js
@@ -1,3 +1,5 @@
+const jQuery = require('jquery');
+
 /*!
  * @copyright Copyright &copy; Kartik Visweswaran, Krajee.com, 2014 - 2016
  * @version 1.3.4
@@ -2680,7 +2682,7 @@ var datetimepickerFactory = function ($) {
 		define(['jquery', 'jquery-mousewheel'], factory);
 	} else if (typeof exports === 'object') {
 		// Node/CommonJS style for Browserify
-		module.exports = factory(require('jquery'));;
+		module.exports = factory(jQuery);;
 	} else {
 		// Browser globals
 		factory(jQuery);
@@ -2702,7 +2704,7 @@ var datetimepickerFactory = function ($) {
         define(['jquery'], factory);
     } else if (typeof exports === 'object') {
         // Node/CommonJS style for Browserify
-        module.exports = factory;
+      module.exports = factory(jQuery);
     } else {
         // Browser globals
         factory(jQuery);
diff --git a/node_modules/jquery-datetimepicker/jquery.datetimepicker.js b/node_modules/jquery-datetimepicker/jquery.datetimepicker.js
index 2c73ed5..74e850f 100644
--- a/node_modules/jquery-datetimepicker/jquery.datetimepicker.js
+++ b/node_modules/jquery-datetimepicker/jquery.datetimepicker.js
@@ -2374,7 +2374,7 @@ var datetimepickerFactory = function ($) {
 					}
 				}
 
-				if ($.type(options.mask) === 'string') {
+				if (typeof options.mask === 'string') {
 					if (!isValidValue(options.mask, input.val())) {
 						input.val(options.mask.replace(/[0-9]/g, '_'));
 						setCaretPos(input[0], 0);
@@ -2607,7 +2607,7 @@ var datetimepickerFactory = function ($) {
 		this.each(function () {
 			var datetimepicker = $(this).data('xdsoft_datetimepicker'), $input;
 			if (datetimepicker) {
-				if ($.type(opt) === 'string') {
+				if (typeof opt === 'string') {
 					switch (opt) {
 						case 'show':
 							$(this).select().focus();
@@ -2644,7 +2644,7 @@ var datetimepickerFactory = function ($) {
 				}
 				return 0;
 			}
-			if ($.type(opt) !== 'string') {
+			if (typeof opt !== 'string') {
 				if (!options.lazyInit || options.open || options.inline) {
 					createDateTimePicker($(this));
 				} else {
@@ -2677,5 +2677,3 @@ var datetimepickerFactory = function ($) {
 		factory(jQuery);
 	}
 }(datetimepickerFactory));
-
-
