diff --git a/node_modules/jquery-datetimepicker/build/jquery.datetimepicker.full.js b/node_modules/jquery-datetimepicker/build/jquery.datetimepicker.full.js
index 7245608..4fb2d51 100644
--- a/node_modules/jquery-datetimepicker/build/jquery.datetimepicker.full.js
+++ b/node_modules/jquery-datetimepicker/build/jquery.datetimepicker.full.js
@@ -1,3 +1,5 @@
+const jQuery = require('jquery');
+
 /*!
  * @copyright Copyright &copy; Kartik Visweswaran, Krajee.com, 2014 - 2016
  * @version 1.3.4
@@ -1124,15 +1126,15 @@ var datetimepickerFactory = function ($) {
 
 				options = $.extend(true, {}, options, _options);
 
-				if (_options.allowTimes && $.isArray(_options.allowTimes) && _options.allowTimes.length) {
+				if (_options.allowTimes && Array.isArray(_options.allowTimes) && _options.allowTimes.length) {
 					options.allowTimes = $.extend(true, [], _options.allowTimes);
 				}
 
-				if (_options.weekends && $.isArray(_options.weekends) && _options.weekends.length) {
+				if (_options.weekends && Array.isArray(_options.weekends) && _options.weekends.length) {
 					options.weekends = $.extend(true, [], _options.weekends);
 				}
 
-				if (_options.allowDates && $.isArray(_options.allowDates) && _options.allowDates.length) {
+				if (_options.allowDates && Array.isArray(_options.allowDates) && _options.allowDates.length) {
 					options.allowDates = $.extend(true, [], _options.allowDates);
 				}
 
@@ -1140,7 +1142,7 @@ var datetimepickerFactory = function ($) {
 					options.allowDateRe = new RegExp(_options.allowDateRe);
 				}
 
-				if (_options.highlightedDates && $.isArray(_options.highlightedDates) && _options.highlightedDates.length) {
+				if (_options.highlightedDates && Array.isArray(_options.highlightedDates) && _options.highlightedDates.length) {
 					$.each(_options.highlightedDates, function (index, value) {
 						var splitData = $.map(value.split(','), $.trim),
 							exDesc,
@@ -1159,7 +1161,7 @@ var datetimepickerFactory = function ($) {
 					options.highlightedDates = $.extend(true, [], highlightedDates);
 				}
 
-				if (_options.highlightedPeriods && $.isArray(_options.highlightedPeriods) && _options.highlightedPeriods.length) {
+				if (_options.highlightedPeriods && Array.isArray(_options.highlightedPeriods) && _options.highlightedPeriods.length) {
 					highlightedDates = $.extend(true, [], options.highlightedDates);
 					$.each(_options.highlightedPeriods, function (index, value) {
 						var dateTest, // start date
@@ -1169,7 +1171,7 @@ var datetimepickerFactory = function ($) {
 							keyDate,
 							exDesc,
 							style;
-						if ($.isArray(value)) {
+						if (Array.isArray(value)) {
 							dateTest = value[0];
 							dateEnd = value[1];
 							desc = value[2];
@@ -1201,11 +1203,11 @@ var datetimepickerFactory = function ($) {
 					options.highlightedDates = $.extend(true, [], highlightedDates);
 				}
 
-				if (_options.disabledDates && $.isArray(_options.disabledDates) && _options.disabledDates.length) {
+				if (_options.disabledDates && Array.isArray(_options.disabledDates) && _options.disabledDates.length) {
 					options.disabledDates = $.extend(true, [], _options.disabledDates);
 				}
 
-				if (_options.disabledWeekDays && $.isArray(_options.disabledWeekDays) && _options.disabledWeekDays.length) {
+				if (_options.disabledWeekDays && Array.isArray(_options.disabledWeekDays) && _options.disabledWeekDays.length) {
 					options.disabledWeekDays = $.extend(true, [], _options.disabledWeekDays);
 				}
 
@@ -1876,7 +1878,7 @@ var datetimepickerFactory = function ($) {
 
 						line_time = function line_time(h, m) {
 							var now = _xdsoft_datetime.now(), current_time,
-								isALlowTimesInit = options.allowTimes && $.isArray(options.allowTimes) && options.allowTimes.length;
+								isALlowTimesInit = options.allowTimes && Array.isArray(options.allowTimes) && options.allowTimes.length;
 							now.setHours(h);
 							h = parseInt(now.getHours(), 10);
 							now.setMinutes(m);
@@ -1907,7 +1909,7 @@ var datetimepickerFactory = function ($) {
 							time += '<div class="xdsoft_time ' + classes.join(' ') + '" data-hour="' + h + '" data-minute="' + m + '">' + dateHelper.formatDate(now, options.formatTime) + '</div>';
 						};
 
-						if (!options.allowTimes || !$.isArray(options.allowTimes) || !options.allowTimes.length) {
+						if (!options.allowTimes || !Array.isArray(options.allowTimes) || !options.allowTimes.length) {
 							for (i = 0, j = 0; i < (options.hours12 ? 12 : 24); i += 1) {
 								for (j = 0; j < 60; j += options.step) {
 								        var currentMinutesOfDay = i * 60 + j;
@@ -2383,7 +2385,7 @@ var datetimepickerFactory = function ($) {
 					}
 				}
 
-				if ($.type(options.mask) === 'string') {
+				if (typeof options === 'string') {
 					if (!isValidValue(options.mask, input.val())) {
 						input.val(options.mask.replace(/[0-9]/g, '_'));
 						setCaretPos(input[0], 0);
@@ -2616,7 +2618,7 @@ var datetimepickerFactory = function ($) {
 		this.each(function () {
 			var datetimepicker = $(this).data('xdsoft_datetimepicker'), $input;
 			if (datetimepicker) {
-				if ($.type(opt) === 'string') {
+				if (typeof opt === 'string') {
 					switch (opt) {
 						case 'show':
 							$(this).select().focus();
@@ -2653,7 +2655,7 @@ var datetimepickerFactory = function ($) {
 				}
 				return 0;
 			}
-			if ($.type(opt) !== 'string') {
+			if (typeof opt !== 'string') {
 				if (!options.lazyInit || options.open || options.inline) {
 					createDateTimePicker($(this));
 				} else {
@@ -2680,7 +2682,7 @@ var datetimepickerFactory = function ($) {
 		define(['jquery', 'jquery-mousewheel'], factory);
 	} else if (typeof exports === 'object') {
 		// Node/CommonJS style for Browserify
-		module.exports = factory(require('jquery'));;
+		module.exports = factory(jQuery);;
 	} else {
 		// Browser globals
 		factory(jQuery);
@@ -2702,7 +2704,7 @@ var datetimepickerFactory = function ($) {
         define(['jquery'], factory);
     } else if (typeof exports === 'object') {
         // Node/CommonJS style for Browserify
-        module.exports = factory;
+      module.exports = factory(jQuery);
     } else {
         // Browser globals
         factory(jQuery);
diff --git a/node_modules/jquery-datetimepicker/jquery.datetimepicker.js b/node_modules/jquery-datetimepicker/jquery.datetimepicker.js
index 2c73ed5..0ec440a 100644
--- a/node_modules/jquery-datetimepicker/jquery.datetimepicker.js
+++ b/node_modules/jquery-datetimepicker/jquery.datetimepicker.js
@@ -1115,15 +1115,15 @@ var datetimepickerFactory = function ($) {
 
 				options = $.extend(true, {}, options, _options);
 
-				if (_options.allowTimes && $.isArray(_options.allowTimes) && _options.allowTimes.length) {
+				if (_options.allowTimes && Array.isArray(_options.allowTimes) && _options.allowTimes.length) {
 					options.allowTimes = $.extend(true, [], _options.allowTimes);
 				}
 
-				if (_options.weekends && $.isArray(_options.weekends) && _options.weekends.length) {
+				if (_options.weekends && Array.isArray(_options.weekends) && _options.weekends.length) {
 					options.weekends = $.extend(true, [], _options.weekends);
 				}
 
-				if (_options.allowDates && $.isArray(_options.allowDates) && _options.allowDates.length) {
+				if (_options.allowDates && Array.isArray(_options.allowDates) && _options.allowDates.length) {
 					options.allowDates = $.extend(true, [], _options.allowDates);
 				}
 
@@ -1131,7 +1131,7 @@ var datetimepickerFactory = function ($) {
 					options.allowDateRe = new RegExp(_options.allowDateRe);
 				}
 
-				if (_options.highlightedDates && $.isArray(_options.highlightedDates) && _options.highlightedDates.length) {
+				if (_options.highlightedDates && Array.isArray(_options.highlightedDates) && _options.highlightedDates.length) {
 					$.each(_options.highlightedDates, function (index, value) {
 						var splitData = $.map(value.split(','), $.trim),
 							exDesc,
@@ -1150,7 +1150,7 @@ var datetimepickerFactory = function ($) {
 					options.highlightedDates = $.extend(true, [], highlightedDates);
 				}
 
-				if (_options.highlightedPeriods && $.isArray(_options.highlightedPeriods) && _options.highlightedPeriods.length) {
+				if (_options.highlightedPeriods && Array.isArray(_options.highlightedPeriods) && _options.highlightedPeriods.length) {
 					highlightedDates = $.extend(true, [], options.highlightedDates);
 					$.each(_options.highlightedPeriods, function (index, value) {
 						var dateTest, // start date
@@ -1160,7 +1160,7 @@ var datetimepickerFactory = function ($) {
 							keyDate,
 							exDesc,
 							style;
-						if ($.isArray(value)) {
+						if (Array.isArray(value)) {
 							dateTest = value[0];
 							dateEnd = value[1];
 							desc = value[2];
@@ -1192,11 +1192,11 @@ var datetimepickerFactory = function ($) {
 					options.highlightedDates = $.extend(true, [], highlightedDates);
 				}
 
-				if (_options.disabledDates && $.isArray(_options.disabledDates) && _options.disabledDates.length) {
+				if (_options.disabledDates && Array.isArray(_options.disabledDates) && _options.disabledDates.length) {
 					options.disabledDates = $.extend(true, [], _options.disabledDates);
 				}
 
-				if (_options.disabledWeekDays && $.isArray(_options.disabledWeekDays) && _options.disabledWeekDays.length) {
+				if (_options.disabledWeekDays && Array.isArray(_options.disabledWeekDays) && _options.disabledWeekDays.length) {
 					options.disabledWeekDays = $.extend(true, [], _options.disabledWeekDays);
 				}
 
@@ -1867,7 +1867,7 @@ var datetimepickerFactory = function ($) {
 
 						line_time = function line_time(h, m) {
 							var now = _xdsoft_datetime.now(), current_time,
-								isALlowTimesInit = options.allowTimes && $.isArray(options.allowTimes) && options.allowTimes.length;
+								isALlowTimesInit = options.allowTimes && Array.isArray(options.allowTimes) && options.allowTimes.length;
 							now.setHours(h);
 							h = parseInt(now.getHours(), 10);
 							now.setMinutes(m);
@@ -1898,7 +1898,7 @@ var datetimepickerFactory = function ($) {
 							time += '<div class="xdsoft_time ' + classes.join(' ') + '" data-hour="' + h + '" data-minute="' + m + '">' + dateHelper.formatDate(now, options.formatTime) + '</div>';
 						};
 
-						if (!options.allowTimes || !$.isArray(options.allowTimes) || !options.allowTimes.length) {
+						if (!options.allowTimes || !Array.isArray(options.allowTimes) || !options.allowTimes.length) {
 							for (i = 0, j = 0; i < (options.hours12 ? 12 : 24); i += 1) {
 								for (j = 0; j < 60; j += options.step) {
 								        var currentMinutesOfDay = i * 60 + j;
@@ -2374,7 +2374,7 @@ var datetimepickerFactory = function ($) {
 					}
 				}
 
-				if ($.type(options.mask) === 'string') {
+				if (typeof options.mask === 'string') {
 					if (!isValidValue(options.mask, input.val())) {
 						input.val(options.mask.replace(/[0-9]/g, '_'));
 						setCaretPos(input[0], 0);
@@ -2607,7 +2607,7 @@ var datetimepickerFactory = function ($) {
 		this.each(function () {
 			var datetimepicker = $(this).data('xdsoft_datetimepicker'), $input;
 			if (datetimepicker) {
-				if ($.type(opt) === 'string') {
+				if (typeof opt === 'string') {
 					switch (opt) {
 						case 'show':
 							$(this).select().focus();
@@ -2644,7 +2644,7 @@ var datetimepickerFactory = function ($) {
 				}
 				return 0;
 			}
-			if ($.type(opt) !== 'string') {
+			if (typeof opt !== 'string') {
 				if (!options.lazyInit || options.open || options.inline) {
 					createDateTimePicker($(this));
 				} else {
@@ -2677,5 +2677,3 @@ var datetimepickerFactory = function ($) {
 		factory(jQuery);
 	}
 }(datetimepickerFactory));
-
-
