diff --git a/node_modules/walkjs/lib/defaults.js b/node_modules/walkjs/lib/defaults.js
index 6817a07..014cd0b 100644
--- a/node_modules/walkjs/lib/defaults.js
+++ b/node_modules/walkjs/lib/defaults.js
@@ -27,7 +27,7 @@ function buildDefaultContext(config) {
             postVisit: []
         },
         config: {
-            trackExecutedCallbacks: true,
+            trackExecutedCallbacks: config.trackExecutedCallbacks === undefined ? true : config.trackExecutedCallbacks,
             visitationRegister: config.visitationRegister || new SetVisitationRegister(),
             traversalMode: (_c = config.traversalMode) !== null && _c !== void 0 ? _c : 'depth',
             graphMode: (_d = config.graphMode) !== null && _d !== void 0 ? _d : 'finiteTree',
