--- ./lock/src/Util/Loop.php.orig	2025-07-28 11:32:08.000000000 +0200
+++ ./lock/src/Util/Loop.php	2025-10-23 12:52:24.722848613 +0200
@@ -64,7 +64,8 @@
         $deadlineTs = microtime(true) + $timeout;
 
         $result = null;
-        for ($i = 0; $this->looping && microtime(true) < $deadlineTs; ++$i) { // @phpstan-ignore booleanAnd.leftAlwaysTrue
+        $i = 0;
+        do {
             $result = $code();
             if (!$this->looping) { // @phpstan-ignore booleanNot.alwaysFalse
                 // The $code callback has called $this->end() and the lock has been acquired.
@@ -89,7 +90,8 @@
             $usecToSleep = min($usecRemaining, random_int((int) $min, (int) $max));
 
             usleep($usecToSleep);
-        }
+            ++$i;
+        } while ($this->looping && microtime(true) < $deadlineTs);
 
         throw LockAcquireTimeoutException::create($timeout);
     }
