--- symfony/string/Slugger/AsciiSlugger.php~	2025-09-11 16:36:48.000000000 +0200
+++ symfony/string/Slugger/AsciiSlugger.php	2025-10-28 11:44:26.005816926 +0100
@@ -101,7 +101,7 @@
         return $new;
     }
 
-    public function slug(string $string, string $separator = '-', ?string $locale = null): AbstractUnicodeString
+    public function transliterate(string $string, ?string $locale = null): AbstractUnicodeString
     {
         $locale ??= $this->defaultLocale;
 
@@ -143,7 +143,12 @@
             }
         }
 
-        return $unicodeString
+        return $unicodeString;
+    }
+
+    public function slug(string $string, string $separator = '-', ?string $locale = null): AbstractUnicodeString
+    {
+        return $this->transliterate($string, $locale)
             ->replaceMatches('/[^A-Za-z0-9]++/', $separator)
             ->trim($separator)
         ;
