--- horde-imap-client/lib/Horde/Imap/Client/Socket/Connection/Socket.php.orig	2023-09-11 15:51:45.000000000 +0200
+++ horde-imap-client/lib/Horde/Imap/Client/Socket/Connection/Socket.php	2024-03-07 20:37:35.335012222 +0100
@@ -93,14 +93,33 @@
             $data = $data->stream;
         }
 
+        if (is_callable($this->_progressCallback)) {
+            $size = -1;
+            if (fseek($data, 0, SEEK_END) === 0) {
+                $size = ftell($data);
+            }
+            $chunkSize = $this->_chunkSize ?? 8192;
+            $pos = 0;
+            $oldPos = $pos;
+            $mode = 'write';
+            $this->_progressCallback($mode, $pos, $size);
+        }
+
         if (rewind($data)) {
             $success = true;
             while (!feof($data)) {
                 if ((($read_data = fread($data, 8192)) === false) ||
-                    (fwrite($this->_stream, $read_data) === false)) {
+                    (($written = fwrite($this->_stream, $read_data)) === false)) {
                     $success = false;
                     break;
                 }
+                if (is_callable($this->_progressCallback)) {
+                    $pos += $written;
+                    if (feof($data) || $pos >= $oldPos + $chunkSize) {
+                        $this->_progressCallback($mode, $pos, $size);
+                        $oldPos = $pos;
+                    }
+                }
             }
         }
 
